discovery-infra/test_infra/tools/terraform_utils.py

```diff
@@ -36,6 +36,7 @@ def change_variables(self, variables, refresh=True):
         self.apply(refresh=refresh)
 
     def get_state(self):
+        self.tf.read_state_file("terraform.tfstate")
```

Comment:
Add `STATE_FILE` constant to class scope and reuse it both here and in `__init__`

Reply:
(just like `VAR_FILE`)